src/Standards/Generic/Tests/CodeAnalysis/EmptyStatementUnitTest.inc

```diff
@@ -69,4 +69,18 @@ try {
     // TODO: Handle this exception later :-)
 }
 
-if (true) {} elseif (false) {}
\ No newline at end of file
+if (true) {} elseif (false) {}
+
+// phpcs:set Generic.CodeAnalysis.EmptyStatement allowComments true
+
+if ($foo) {
+    // Just a comment
+} elseif ($bar) {
+    /*
+     * Yet another comment
+     */
+} elseif ($baz) {
+    // phpcs:set Generic.CodeAnalysis.EmptyStatement somethingMadeUp true
+} else {
+
+}
```

Comment:
You should probably reset the changed property to the default at the end of this test to prevent issues with future test cases being added.